docs/sphinx/source/conf.py

```diff
@@ -221,6 +221,8 @@ def setup(app):
     app.add_stylesheet("no_scrollbars.css")
     # Override footnote callout CSS to be normal text instead of superscript
     app.add_stylesheet("no_reference_superscript.css")
+    # In-line links to references as numbers in brackets.
+    app.add_stylesheet("bracketed_reference_number.css")
```

Comment:
Found this in the logs on RTD, would be nice to get rid of it before merging:

```
/home/docs/checkouts/readthedocs.org/user_builds/pvlib-python/checkouts/1075/docs/sphinx/source/conf.py:221:
RemovedInSphinx40Warning: The app.add_stylesheet() is deprecated. Please use app.add_css_file() instead.
```

Reply:
updated